cmd/comparator-rest/docs/openapi.yaml

```diff
@@ -218,11 +195,17 @@ components:
             duration:
               type: integer
               description: Duration (in seconds) for which this authorization will remain valid.
-    Comparison:
+    ComparisonRequest:
       type: object
       properties:
         op:
           $ref: "#/components/schemas/Operator"
+    Comparison:
+      description: TODO - "comparison" does not sound apt as a name
```

Comment:
This description needs to be fixed: `Result of the comparison.`

Reply:
done